tools/portfolio/verify-portfolio.sh

```
@@ -0,0 +1,249 @@
+    fail aprove-ceta "ceta not on PATH"
+    return
+  fi
+  dir="$tmp_root/aprove-ceta"
+  mkdir -p "$dir"
+  cat > "$dir/system.trs" <<'TRS'
+(VAR x)
+(RULES
+  f(x) -> x
+)
+TRS
+  if ! out="$(aprove -m wst -p cpf -C ceta "$dir/system.trs" > "$dir/raw.cpf" 2>&1)"; then
```

Shellcheck (0.11.0)

[warning] This command substitution will be empty because the command's output gets redirected away.

(SC2327)